scripts/python-maintainer/generate_python_package.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
```

Comment:
can you add 
# SPDX-License-Identifier: MIT
# 2016 Alexander Couzens <lynxis@fe80.eu>

also add yourself if you've changed more here.